ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/datepicker/v1/datepicker/clientlibs/site/css/datepickerwidget.css

```diff
@@ -154,7 +154,7 @@
 
 .cmp-adaptiveform-datepicker__calendar-icon {
     position: absolute;
-    top: 45px;
+    top: 50%;
```

Comment:
Please add test case for this, the new cypress version also support such use-cases